### PR TITLE
Add docs for 1809

### PR DESCRIPTION
**Pull request by shirhatti**

### CODE REVIEW

File: iis/get-started/whats-new-in-iis-10-version-1809/new-features-introduced-in-iis-10-1809.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+IIS 10.0 version 1809 is a cumulative update if you are upgrading from Windows Server 2016 and includes all the features introduced in version 1709 and version 1803.
+
+## New binding flags
+
+We introduced support for OCSP stapling in IIS 7.0 and support for HTTP/2 in IIS 10.0, but they were only controlled on a system-wide level. In IIS 10.0 version 1809 and later you can control both OCSP Stapling and HTTP/2 on a per-binding basis.
+
+## New compression API
+
+The static and dynamic compression modules in IIS provide support for pluggable compression scheme providers. As part of IIS 10.0 version 1809, we've introduced a [new compression API](https://msdn.microsoft.com/library/mt846747(v=vs.90).aspx) that compression schemes can optionally implement to better support compressing of streaming data. The new [IIS Compression module](https://docs.microsoft.com/en-us/iis/extensions/iis-compression/iis-compression-overview) that adds a new gzip compression scheme (with better for streaming data) and a brotli compression scheme make use of this API.
```
Comment (shirhatti) [inline iis/get-started/whats-new-in-iis-10-version-1809/new-features-introduced-in-iis-10-1809.md]: brotli -> Brotli

Comment (shirhatti) [reply iis/get-started/whats-new-in-iis-10-version-1809/new-features-introduced-in-iis-10-1809.md]: > make use of this API.

both make use of this API.

Comment (EatonZ) [reply iis/get-started/whats-new-in-iis-10-version-1809/new-features-introduced-in-iis-10-1809.md]: I think (with better for streaming data) should be (with better **support** for streaming data).

**Comment by mairaw** [comment]
@shirhatti this now has merge conflicts

**Comment by Rick-Anderson** [comment]
@shirhatti your **_really_** important PR now has merge conflicts I can't fix. If you can fix I'll S&M and merge to live.

**Comment by Rick-Anderson** [comment]
@wadepickett can you fix the merge conflicts here?

File: iis/TOC.md [hunk 2]
```diff
@@ -9,6 +9,8 @@
 ## [Access Protection](overview/access-protection.md)
 
 # Get Started
+## What’s New in IIS 10.0, version 1809?
+### [New Features Introduced in IIS 10.0, version 1809](get-started/whats-new-in-iis-10-version-1809/new-features-introduced-in-iis-10-1809.md)
```
Comment (Rick-Anderson) [inline iis/TOC.md:13]:
```suggestion
## What’s New in IIS 10.0, version 1809?
### [New Features Introduced in IIS 10.0, version 1809](get-started/whats-new-in-iis-10-version-1809/new-features-introduced-in-iis-10-1809.md)
```

**Comment by Rick-Anderson** [comment]
Add the following to  line 11
```
# Get Started
## What’s New in IIS 10.0, version 1809?
### [New Features Introduced in IIS 10.0, version 1809](get-started/whats-new-in-iis-10-version-1809/new-features-introduced-in-iis-10-1809.md)
```

File: iis/get-started/index.md [hunk 3]
```diff
@@ -1,3 +1,3 @@
 ---
-redirect_url: /iis/get-started/whats-new-in-iis-10-version-1709
+redirect_url: /iis/get-started/whats-new-in-iis-10-version-1809
```
Comment (Rick-Anderson) [inline iis/get-started/index.md:2]:
```suggestion
redirect_url: /iis/get-started/whats-new-in-iis-10-version-1709
```

**Comment by Rick-Anderson** [comment]
Possibly revert 
image

**Pull request merged**